backend/src/main/java/ai/verta/modeldb/dataset/DatasetDAORdbImpl.java

```diff
@@ -372,12 +372,12 @@ public Boolean deleteDatasets(List<String> datasetIds) throws InvalidProtocolBuf
   @Override
   public Boolean setUpdateTime(List<String> datasetIds, long timestamp) {
     try (Session session = ModelDBHibernateUtil.getSessionFactory().openSession()) {
-      Transaction transaction = session.beginTransaction();
+      //Transaction transaction = session.beginTransaction();
       Query query = session.createQuery(UPDATE_TIME_QUERY);
       query.setParameter("timestamp", timestamp);
       query.setParameterList("ids", datasetIds);
       int result = query.executeUpdate();
-      transaction.commit();
+      //transaction.commit();
```

Comment:
this transaction is required because we are doing `Update` operation here without transaction update is not work.

Reply:
Most databases (our included) should support auto-commit of changes. That is the recommended approach if you're not going to make a lot of related requests that need consistency between them.

Reply:
Should auto-commit be enabled somehow in hibernate settings?

Reply:
Maybe. I know it's set in our database itself.